src/ccmain/ltrresultiterator.cpp

```diff
@@ -310,11 +308,7 @@ char *LTRResultIterator::WordTruthUTF8Text() const {
   if (!HasTruthString()) {
```

Comment:
What if we change this function return type to std::string?
```
char *LTRResultIterator::WordTruthUTF8Text() const {
```
to
```
std::string LTRResultIterator::WordTruthUTF8Text() const {
```
and also all calling sites?